apps/src/music/player/MusicLibrary.ts

```diff
@@ -2,7 +2,7 @@ export default interface MusicLibrary {
   groups: FolderGroup[];
```

Comment:
Could you add an optional `type?: 'kit'` param to the `SoundFolder` interface to match the new library JSON structure? That'll be handy when down the road when the view get converted to TS for example.

Reply:
Good call.